Web/stockdogweb/src/stories/index.js

```diff
@@ -37,20 +40,40 @@ var listingProps = {
 
 storiesOf('Listing', module)
    .add('one listing', () =>
-      <Listing {...listingProps}/>);
+      <Listing {...listingProps} />);
 
 storiesOf('Graph', module)
-   .add('loading', () => 
+   .add('loading', () =>
       <Graph isLoading={true} />)
-   .add('loaded', () => 
-      <Graph isLoading={false} 
+   .add('loaded', () =>
+      <Graph isLoading={false}
          labels={["1/13/2018", "1/14/2018", "1/15/2018",
-                  "1/16/2018", "1/17/2018", "1/18/2018"]} 
-         data={[14.21, 23.21, 20.53, 19.23, 15.67, 16.23]} 
+            "1/16/2018", "1/17/2018", "1/18/2018"]}
+         data={[14.21, 23.21, 20.53, 19.23, 15.67, 16.23]}
       />
    );
 
 
+const navbarLinks = [
+   {
+      title: "Month League",
+      location: "/league/monthLeague"
```

Comment:
Are these generated based on people's leagues? How will you know the location for each league when it's based on the person? Does it work like an api endpoint but for routes?

Reply:
These are just placeholders for now. I'm thinking the actual location will be the league's ID once we have those in the api/db.